sklearn/feature_extraction/text.py

```diff
@@ -1346,7 +1357,8 @@ class TfidfTransformer(TransformerMixin, BaseEstimator):
     ----------
     idf_ : array, shape (n_features)
         The inverse document frequency (IDF) vector; only defined
-        if  ``use_idf`` is True.
+        if  ``use_idf`` is True.      
```

Comment:
I think we need an empty line before `versionadded`

Reply:
Adding this in.